packages/ndk-multilib/build.sh

```diff
@@ -37,11 +37,59 @@ prepare_libs() {
 	fi
 
 	cp $LIBATOMIC/libatomic.a $TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX/$SUFFIX/lib/
+
+	cp $LIBATOMIC/libunwind.a $TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX/$SUFFIX/lib/
+}
+
+add_cross_compiler_rt() {
+	RT_PREFIX=$NDK/toolchains/llvm/prebuilt/linux-x86_64/lib64/clang/*/lib/linux
+	RT_OPT_DIR=$TERMUX_PREFIX/opt/ndk-multilib/cross-compiler-rt
+	mkdir -p $TERMUX_PKG_MASSAGEDIR/$RT_OPT_DIR
+	cp $RT_PREFIX/* $TERMUX_PKG_MASSAGEDIR/$RT_OPT_DIR || true
 }
 
 termux_step_extract_into_massagedir() {
 	prepare_libs "arm" "arm-linux-androideabi"
 	prepare_libs "arm64" "aarch64-linux-android"
 	prepare_libs "x86" "i686-linux-android"
 	prepare_libs "x86_64" "x86_64-linux-android"
+	add_cross_compiler_rt
+}
+
+termux_step_create_debscripts() {
+	LIBLLVM_VERSION=$(. $TERMUX_SCRIPTDIR/packages/libllvm/build.sh; echo $TERMUX_PKG_VERSION)
+	RT_PATH=$TERMUX_PREFIX/lib/clang/$LIBLLVM_VERSION/lib/android
+	# Install the symlinks.
+	cat <<- POSTINST_EOF > ./postinst
+	#!$TERMUX_PREFIX/bin/sh
+
+	echo "Installing symlinks to $RT_PATH..."
+
+	mkdir -p $RT_PATH
+	find $RT_OPT_DIR -type f ! -name "libclang_rt*\$(uname -m)*" -exec ln -sfr "{}" $RT_PATH \;
```

Comment:
This won't work in some environments. In my 32-bit ARM phone `uname -m` prints `armv7l`.

For an alternative way, see:

https://github.com/termux/termux-packages/blob/b3d94b5399078aa07f33294ca7bc40343a3fb6a7/build-package.sh#L285-L286

Reply:
OK, fixed.